src/installation/guides/raspberry-pi.md

```diff
@@ -0,0 +1,201 @@
+file.
+
+Insert the SD card and test that the Raspberry PI boots correctly.
+
+## First Boot
+
+The `root` password is `voidlinux`.
+
+### Set the system time
+
+Before it is possible to install or upgrade packages, it is necessary to set the
+clock, or you will see HTTPS certificate errors. The Raspberry Pi does not have
+a battery-backed clock, so you must set the time manually.
+
+Install and enable [an NTP server](../../config/date-time.md#ntp) and wait for
+the date and time to be set.
+
+??? However, if you still encounter SSL related errors or the time is still not
+correct, it might be worth to reconfigure package chrony (`xbps-reconfigure -f
+chrony`) because some images tend to not create chrony user and group, often
+required by ntpd. ???
+
+### Enabling hardware RNG device
+
+By default the
+[HWRNG](https://en.wikipedia.org/wiki/Hardware_random_number_generator) device
+is not usedm which may result in the random devices taking long to seed on boot
+(annoying if you want to start `sshd` and expect to be able to connect
+immediately).
+
+First, add the RPi's HWRNG kernel module (`bcm2835_rng`) to a file in
+`/etc/modules-load.d`, such as `/etc/modules-load.d/rng.conf`.
+
+Then install `rng-tools` and
+[enable](../../config/services/index.md#enabling-services) the `rngd` service so
+that the kernel's randomness will be seeded by the `hwrng` device.
+
+## Applications
+
+### X11
+
+At a minimum, you will need to install `xorg-minimal` and `xf86-video-fbturbo`.
+Client applications are available in `xorg-apps`, and X fonts are available in
+`xorg-fonts`.
+
+Make sure the user running X is part of the groups `audio` and `video`.
+
+## Hardware
+
+### Audio
+
+To get the soundchip to work, add `dtparam=audio=on` to `/boot/config.txt`.
+
+### Serial
+
+To enable serial console logins:
+
+```
+# mkdir ~/rootfs
+# mount /dev/sdX2 ~/rootfs
+# ln -s /etc/sv/agetty-ttyAMA0 ~/rootfs/etc/runit/runsvdir/default
+# umount ~/rootfs
+```
+
+Notes:
+
+- See `/boot/cmdline.txt` for start-up configuration of serial port (device and
+   baud).
+- See `/etc/securetty` for interfaces that allow root login, ttyAMA0 should
+   already be listed.
+
+### I2C
+
+To enable [I2C](https://en.wikipedia.org/wiki/I%C2%B2C), add
+`device_tree_param=i2c_arm=on` to `/boot/config.txt`, and
+`bcm2708.vc_i2c_override=1` at the end of `/boot/cmdline.txt`.
+
+If the latter doesn't exist, create the directory `/etc/modules-load.d/`, and in
+that directory, create the file `i2c.conf` with a line `i2c-dev`. Then reboot.
+
+Install the `i2c-tools` package and run `i2cdetect -l`. It should show:
+
+```
+i2c-1i2c          bcm2835 I2C adapter                 I2C adapter
+```
+
+## Miscellaneous
+
+### DKMS
+
+Building kernel modules (like `wireguard` or `zfs`) should generally work by
+installing the package in question, but due to current limitations, you need to
+install the `rpi-kernel-headers` package manually as well.
```

Comment:
This should be mentioned elsewhere. Possibly in the kernel section: "the kernel packages also have their own header packages: `rpi-kernel-headers`..."

Reply:
i've added that sentence to the end of "The `rpi-kernel` packages ..." para; let me know if that's not what you meant.